docs/gen_paramters_doc.py

```diff
@@ -40,6 +40,18 @@
     Current quality flags are: :ref:`bottle <Bottle Quality Codes>`, :ref:`water <Water Quality Codes>`, :ref:`ctd <CTD Quality Codes>`.
     See the :ref:`Quality Codes` section for more information
 
+Parameter Error
+---------------
+Any parameter which numeric values MAY have an associated error column which takes the form:
```

Comment:
Looks good! 
One small fix
this : Any parameter which numeric values
should be : Any parameter with numeric values
which -> with